src/v2/Apps/Order/Routes/Review/index.tsx

```diff
@@ -122,13 +124,34 @@ export class ReviewRoute extends Component<ReviewProps> {
             }
           })
       } else {
-        this.props.order.conversation && !this.props.isEigen
-          ? this.props.router.push(
-              `/user/conversations/${this.props.order.conversation.internalID}`
-            )
-          : this.props.router.push(
-              `/orders/${this.props.order.internalID}/status`
-            )
+        const { order, router, user, isEigen } = this.props
+
+        if (!userHasLabFeature(user, "Make Offer On All Eligible Artworks")) {
+          return order.conversation && !isEigen
+            ? router.push(
+                `/user/conversations/${order.conversation.internalID}`
+              )
+            : router.push(`/orders/${order.internalID}/status`)
+        }
+        // Buy-mode order redirects to the status page
+        if (order.mode !== "OFFER") {
+          return router.push(`/orders/${order.internalID}/status`)
+        }
+        // Make offer in inquiry redirects to the conversation page
+        if (order.source === "inquiry" && !isEigen) {
```

Comment:
Looking again at it, should Eigen redirect to /status here?

Reply:
Update: I tried MOI on Eigen and it goes to /status

Reply:
The app’s checkout flow works on a webview inside a modal on Eigen. There seems to be difficult (or impossible?) to close the modal window from inside a webview, so that’s why we currently take the user to the congrats page (/status) instead of the conversation on MOI. See https://github.com/artsy/force/pull/7974.